data/categories.txt

```diff
@@ -1,4 +1,15 @@
 # Category groups referenced in many categories, should be defined before being referenced.
+# Syntax:
+# @   - string that starts with this symbol indicates the beginning of the group translations, which are used for the spotlight search.
+# :   - symbol that is used to separate the name of the language and translation (or group of translated synonyms). 
+#       Language name is located on the left side of ':', and the translation on the right.
+# |   - symbol that is used to separate several translated synonyms.
+# 1-9 - numbers from one to nine, indicate the weight of the translated word for assumptions in search queries. 
```

Comment:
digits in front of a word indicate the number of symbols than need to be typed in a search query to make this word appear in the list of suggestions.